test/apicoverage/image/common/common.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2019 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"os"
+
+	"github.com/knative/pkg/system"
+	pkgWebhook "github.com/knative/pkg/webhook"
+	"github.com/knative/serving/pkg/apis/serving/v1alpha1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+var (
+	ResourceMap = map[schema.GroupVersionKind]pkgWebhook.GenericCRD {
+		v1alpha1.SchemeGroupVersion.WithKind("Revision"):      &v1alpha1.Revision{},
+		v1alpha1.SchemeGroupVersion.WithKind("Configuration"): &v1alpha1.Configuration{},
+		v1alpha1.SchemeGroupVersion.WithKind("Route"):         &v1alpha1.Route{},
+		v1alpha1.SchemeGroupVersion.WithKind("Service"):       &v1alpha1.Service{},
+	}
+	WebhookNamespace = os.Getenv(system.NamespaceEnvKey)
```

Comment:
You should be able to rely on `github.com/knative/pkg/system` to access this?

Reply:
Fixed.